src/test/util/CMakeLists.txt

```diff
@@ -27,3 +27,15 @@ target_link_libraries(test_util
   PUBLIC
     univalue
 )
+
+function(make_bitcoinconsensus_dll_available target)
+  if(TARGET bitcoinconsensus AND WIN32 AND BUILD_SHARED_LIBS)
```

Comment:
Don't linux shared libs have the same problem? Unless there's some rpath magic, I'd expect to see the same copy for Linux (or LD_LIBRARY_PATH).

Reply:
> Unless there's some rpath magic...

CMake does this for us.

Reply:
It might be easy tested, for example, using `-G "Ninja Multi-Config"`.